src/tests/ftest/datamover/dm_posix_preserve_props.py

```diff
@@ -178,7 +178,7 @@ def verify_cont_prop(self, cont, prop_list, api):
         # Make sure each property matches
         for prop_idx, prop in enumerate(prop_list):
             # This one is not set
-            if api == "DFS" and "OID" in prop_list[prop_idx]:
+            if api == "DFS" and ("OID" or "ROOTS") in prop_list[prop_idx]:
```

Comment:
I think `("OID" or "ROOTS") in prop_list[prop_idx]` will always evaluate to:
`"OID" in prop_list[prop_idx]`

```
>>> my_s = "aaa ROOTS bbb"
>>> ("OID" or "ROOTS")
'OID'
>>> ("OID" or "ROOTS") in my_s
False
>>> "ROOTS" in my_s
True
```